src/index.js

```diff
@@ -0,0 +1,85 @@
+// Importing dependancies
+const inquirer = require("inquirer");
+const path = require("path");
+const fs = require("fs");
+const { writeToFile } = require("./utils/utils.js");
+const generateHTMLfile = require("./utils/renderHtml.js");
+
+// Importing Classes
+const { name, id, email } = require("./lib/Employee");
+const Manager = require("./lib/Manager");
+const Engineer = require("./lib/Engineer");
+const Intern = require("./lib/Intern");
+
+// Importing Questions
+const {
+	introQuestions,
+	questions,
+	loopingQuestion,
+} = require("./questions.js");
+
+// Checking to see if the directory and file foor the generated HTML page already exists
+const OUTPUT_DIR = path.resolve(__dirname, "./src/dist");
+// If not, create directory and file
+const outputPath = path.join(OUTPUT_DIR, "team.html");
+
+// initialize function -prepares the application for user interaction and inputs
+const init = async () => {
+	let inProgress = true;
+
+	// Declaration of empty arrays to hold objects for each team member
+	let engineers = [];
```

Comment:
Arrays don't need to be declared as `let` - the value of the variable is never changing, it is still pointing to the container, we are just adding elements to it. It would only need to be declared as `let` if you were planning to completely reassign the value, ie. something like `engineers = 'team'` or `engineers = [ .....some new array ... ]`. If we're pushing in to it, `const` is best.